topic7/src/main/java/controllers/TeacherController.java

```diff
@@ -0,0 +1,52 @@
+package controllers;
+
+import models.Teacher;
+import models.TeacherDao;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+@Controller
+public class TeacherController {
+
+	@Autowired
+	private TeacherDao _TeacherDao;

```

Comment:
By convention the name of the variables needs to be using camelCase (i.e: teacherDao)